openstack/clientconfig/results.go

```diff
@@ -126,6 +126,20 @@ type AuthInfo struct {
 	// DefaultDomain is the domain ID to fall back on if no other domain has
 	// been specified and a domain is required for scope.
 	DefaultDomain string `yaml:"default_domain,omitempty" json:"default_domain,omitempty"`
+
+	// AllowReauth should be set to true if you grant permission for Gophercloud to
+	// cache your credentials in memory, and to allow Gophercloud to attempt to
+	// re-authenticate automatically if/when your token expires.  If you set it to
+	// false, it will not cache these settings, but re-authentication will not be
+	// possible.  This setting defaults to false.
+	//
+	// NOTE: The reauth function will try to re-authenticate endlessly if left
+	// unchecked. The way to limit the number of attempts is to provide a custom
+	// HTTP client to the provider client and provide a transport that implements
+	// the RoundTripper interface and stores the number of failed retries. For an
+	// example of this, see here:
+	// https://github.com/rackspace/rack/blob/1.0.0/auth/clients.go#L311
```

Comment:
This looks to be copied verbatim from https://github.com/gophercloud/gophercloud/blob/513734676e6495f6fec60e7aaf1f86f1ce807428/auth_options.go#L70-L81 which won't work here.

The original comment is out of date (the URL no longer exists). In addition, the text for this comment isn't the right context for the situation in which a developer would be reading this.

Can you please add a descriptive comment about why this option exists in the clientconfig package?

Reply:
Yes, it's copied. 

In the code the value is in the end set into `gophercloud.AuthOptions` and that argument to `openstack.AuthenticatedClient`, so I thought that the comment could be same as in gophercloud AuthOptions as it configures the same functionality (I'm just passing argument further).

I can shorten the comment to this (it explains everything):

```go
// AllowReauth should be set to true if you grant permission for Gophercloud to
// cache your credentials in memory, and to allow Gophercloud to attempt to
// re-authenticate automatically if/when your token expires.  If you set it to
// false, it will not cache these settings, but re-authentication will not be
// possible.  This setting defaults to false.
```

Or would you suggest something else?

Reply:
This looks good to me - thank you!

Reply:
Shortened the commend in 955acf2e8bee58b0659dec6b5e6a6061c7eb8308